kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -211,10 +218,112 @@ exit
     => ReplM claim ReplStatus
 exit = do
     proofs <- allProofs
+    ofile <- Lens.use lensOutputFile
```

Comment:
This function is way too big. How about something along the lines of:
```
  unfinished <- findNonCompletedNodes -- :: m (Claim, TermLike) or something
  let claims = map makeOnePathClaim unfinished
  printToOutput $ foldClaims claims
  if null unfinished
    then return SuccessStop
    else return FailStop
```

Everything in the where clause should be moved to the appropriate function. Of course, it doesn't *have* to follow this idea, you can split it in a different way if it makes more sense.